### PR TITLE
replaces certain unknown race/ethnicity "AND"s with "OR"s for accuracy

### PR DESCRIPTION
**Pull request by benhammondmusic**

### CODE REVIEW

**Comment by benhammondmusic** [comment]
changes:
- TITLE and ALERT on UNKNOWNS MAP
 - ALERT on RACIAL DISPARITY BAR CHART
 
 works on all 3 carousel settings

**Comment by benhammondmusic** [comment]
changes and to or

**Comment by benhammondmusic** [comment]
and or grammar

**Comment by benhammondmusic** [comment]
fixes #1081

**Comment by joshzarrabi** [comment]
I'm still seeing the wrong end of the sentence on the map card.

Screen Shot 2021-08-17 at 5 25 19 PM

**Comment by benhammondmusic** [comment]
Hmm I'll look

**Comment by benhammondmusic** [comment]
@joshzarrabi 
What did you think it should change to? I think this is correct; should we have it read "where EITHER race or ethnicity was unknown" 

It's singular so I think it should be "was" not "were"

**Comment by joshzarrabi** [comment]
@benhammondmusic My bad,  youre totally right

**Comment by joshzarrabi** [comment]
@benhammondmusic I think adding `either` would be good

**Comment by joshzarrabi** [comment]
@benhammondmusic Also I noticed `sex` and `age` are incorrectly picking up the change.
Screen Shot 2021-08-17 at 5 44 37 PM

**Comment by benhammondmusic** [comment]
ah bummer; i thought i'd checked that. will look into it

**Comment by benhammondmusic** [comment]
@joshzarrabi OK updated it to only affect the correct toggle-tab setting

File: frontend/src/reports/TwoVariableReport.tsx [hunk 1]
```diff
@@ -211,6 +211,7 @@ function TwoVariableReport(props: {
           updateFips: (fips: Fips) => void
         ) => (
           <UnknownsMapCard
+            overrideAndWithOr={true}
```
Comment (joshzarrabi) [inline frontend/src/reports/TwoVariableReport.tsx:214]: do we need to make sure this is only true in the case of race_and_ethnicity as well?

Comment (benhammondmusic) [reply frontend/src/reports/TwoVariableReport.tsx:214]: Yes, good catch. I'll fix that

**Review by joshzarrabi — APPROVED** [review]

**Pull request merged**